canonical/juju.is
# Add favicon

## Done

- Add favicon

## QA

- Visit https://juju-is-490.demos.haus/
- See the favicon is an orange square with the juju logo
